presto-main/src/main/java/io/prestosql/sql/planner/iterative/rule/PushAggregationIntoTableScan.java

```diff
@@ -93,12 +88,15 @@ private static boolean allArgumentsAreSimpleReferences(AggregationNode node)
 
     private static boolean hasNoMasks(AggregationNode node)
     {
-        return !node.getAggregations()
+        return node.getAggregations()
                 .values().stream()
-                .map(aggregation -> aggregation.getMask().isPresent())
-                .anyMatch(isMaskPresent -> isMaskPresent);
+                .noneMatch(aggregation -> aggregation.getMask().isPresent());
```

Comment:
This is stylistic improvement, but not a logic change.
It's better to spit it out into "Improve code style" commit.

Reply:
Removed.